MAUI/PDF-Viewer/Add-Remove-Modify-Annotations.md

```diff
@@ -171,4 +171,21 @@ private void OnAnnotationEdited(object sender, AnnotationEventArgs e)
     Debug.WriteLine("The {0} is edited successfully", editedAnnotation.Name);
 }
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+### Adding custom information to a annotation
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to assign custom information to annotations. The `CustomData` property is utilized to store additional information about each annotation instance for reference. However, it's essential to note that these data are solely intended for reference purposes and will not be displayed in the `SfPdfViewer` interface.
+
+The provided code sample illustrates how to set the `CustomData` property for an annotation. Within this code snippet, we set the custom data to represent the created date of the annotation.
```

Comment:
The provided code sample illustrates how to set the `CustomData` property for an annotation. Within this code sample, we set the custom data to represent the created date of the annotation.